### PR TITLE
[Feature] enhanced onboarding with checking for bluetooth 

### PR DESCRIPTION
**Pull request by steins-code**
Add alert to notify users of disabled bluetooth during onboarding. The user has the option to pass the onboarding without enabling bluetooth by tapping on "Later" or the user can directly go to settings to enable bluetooth. The alert pops up only when bluetooth is disabled.

### CODE REVIEW

**Review by inf2381 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 1]
```diff
@@ -198,7 +210,7 @@ final class OnboardingInfoViewController: UIViewController {
 	// MARK: Exposure notifications
 
 	private func askExposureNotificationsPermissions(completion: (() -> Void)?) {
-		if TestEnvironment.shared.isUITesting {
+		guard !TestEnvironment.shared.isUITesting && !exposureManagerActivated else {
```
Comment (inf2381) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:213]: I don't like `TestEnvironment.shared.isUITesting` inside our code but that's not part of this PR

**Review by i336616 — APPROVED** [review]
👍

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift (hunk 1 above)
Comment (i336616) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:213]: Do we have a preferred way to check for UI testing state at the moment?
